vcenter_operator/templates.py

```diff
@@ -88,7 +88,11 @@ def store_default(env, k):
     _SAVED_DEFAULTS[k] = getattr(env, k)
 
 
-class ConfigMapLoader(BaseLoader):
+class PollingLoader(BaseLoader):
+    def poll(self):
+        raise NotImplementedError()
```

Comment:
What do you think about the [`abc`](https://docs.python.org/3/library/abc.html) module? Looks cleaner to me than raising only if the function is called.

Reply:
I think, in this case, the downsides outweigh the upsides. The base-class is used solely in this module and, if I am not mistaken `abc` requires inheritance from ABC (or derived classes), which introduces multiple inheritance, with the risk of conflicting meta classes, should jinja2 ever decide to also introduce them in the BaseLoader.

Reply:
```
Note that the type of ABC is still ABCMeta, therefore inheriting from ABC requires the usual precautions regarding metaclass usage, as multiple inheritance may lead to metaclass conflicts. One may also define an abstract base class by passing the metaclass keyword and using ABCMeta directly, for example:
```
From the linked docs. Looks like there's a way to do it like in python2 with an explicit metaclass.